recipes/conjure-client/meta.yaml

```diff
@@ -0,0 +1,64 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 13 is conviniet
+# when copying and paster from another recipe, but not really needed.
+{% set name = "conjure-client" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2c806c0849d24e16d0e07014c4fe4b17f4fb3480d5f2f34ab329c87ccf5d372d
+
+build:
+  noarch_python: True
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: python setup.py install
```

Comment:
Change this to `python -m pip install --no-deps --ignore-installed .`

Reply:
This change seems to have broken the build. Something about missing the package internal to the repo:
```
import: 'conjure'
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/conjure-client_1529978829790/test_tmp/run_test.py", line 2, in <module>
    import conjure
ImportError: No module named conjure
```

Reply:
@mariusvniekerk any thoughts?